test/fw/ptl/lib/ptl_mom.py

```diff
@@ -696,7 +696,11 @@ def check_mom_bash_version(self):
         ret = self.du.run_cmd(self.hostname, cmd=cmd, sudo=True,
                               as_script=True)
         req_bash_version = "4.2.46"
-        mom_bash_version = ret['out'][0]
+        if len(ret['out']) > 0:
+            mom_bash_version = ret['out'][0]
+        else:
+            # If it fails to get bash version return True
```

Comment:
Comments should explain why, not what the code is doing.  Why are you returning true if we can't get the bash version?